subsystems/sidecar/index.js

```diff
@@ -36,9 +37,9 @@ const {
   SALT, KNOWN_NODES_LIMIT
 } = require('../../constants')
 const { ERR_INTERNAL_ERROR, ERR_PERMISSION_REQUIRED } = require('../../errors')
+const db = HyperDB.rocks(path.join(PLATFORM_DIR, 'hyperdb'), require('../../spec/db'))
```

Comment:
const definition = require('../spec/db') 

and then pass it here 

also does `./hyperdb` need to be added to .gitignore?